<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>analyzeGoldenWindow</class>
 <widget class="QMainWindow" name="analyzeGoldenWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Analyze Golden Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="frameLabel">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>70</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string>Display image</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>270</width>
      <height>480</height>
     </rect>
    </property>
    <property name="title">
     <string>Golden File</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QTableView" name="goldenTableView">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>237</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&amp;Test Category</string>
       </property>
       <property name="buddy">
        <cstring>testCategoryComboBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="testCategoryComboBox">
       <item>
        <property name="text">
         <string>All</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="scoreGroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>269</width>
      <height>93</height>
     </rect>
    </property>
    <property name="title">
     <string>Score</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>frame</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>total</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="frameScoreLabel">
           <property name="text">
            <string>0.0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="totalScoreLabel">
           <property name="text">
            <string>0.0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>270</width>
      <height>480</height>
     </rect>
    </property>
    <property name="title">
     <string>Test File</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTableView" name="candidateTableView">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="baseSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="frameIdxLabel">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string># frame index</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>237</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>&amp;Golden Category</string>
       </property>
       <property name="buddy">
        <cstring>goldenCategoryComboBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="goldenCategoryComboBox">
       <item>
        <property name="text">
         <string>All</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>choose test file</string>
   </property>
  </action>
  <action name="actionChoose_golden_file">
   <property name="text">
    <string>choose golden file</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
